Fix spelning misteaks (#645)
civicrm/civicrm-dev-docs: docs/standards/index.md

Orange light code is code that has the feel that it is wrong and should be refactored. Developers who are looking to contribute to this effort may want to consider doing any amount self-contained work on the following code. 

* Using joins on `civicrm_option_value` rather than using PseudoConstants - this has performance implications
* Removing as much as possible passing by reference to functions.
* Increasing code complexity, this has two issues firstly it increase the fragility of the code and also makes it harder to test
* Wherever `$session = CRM_Core_Session(); $userid = $session->get('UserID');` or very similar these calls should be replaced with `CRM_Core_Session::singleton()->getLoggedInContactID();`
* Replace `CRM_Core_Resources->singleton()->add...` and similar with `Civi::resources()->add...`

> * Clean up messy code, see if code can be refactored and moved into parent classes and make them more generic, and also eliminate any duplicate code

Reviewer: ```suggestion
* Clean up messy code, see if code can be refactored and/or moved into parent classes. Make parent classes more generic and eliminate any duplicate code.
```

* Increase the usage of Doc blocks to help with auto generation of code

> * Move more business logic out from the Forms and API if possible to the BAO level

Reviewer: ```suggestion
* Move more business logic out of the Form/API layers to the BAO level wherever possible.
```
